src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -390,9 +551,24 @@ def make_projects(self, entry):
             project.pop("_type")
             project_shortname = project["project_shortname"]
             translated_project = {
+                "projectTitle": project.get("project_title"),
                 "projectShortname": project_shortname,
-                "laboratory": list(set(project.get("laboratory")) if project.get("laboratory") else [])
+                "laboratory": list(set(project.get("laboratory", [])))
             }
+
+            if self.projects_response:
+                translated_project["projectDescription"] = project.get("project_description", [])
+                translated_project["contributors"] = project.get("contributors", [])
+                translated_project["publications"] = project.get("publications", [])
+
+                for contributor in translated_project['contributors']:
+                    for key in list(contributor.keys()):
+                        contributor[KeywordSearchResponse.to_camel_case(key)] = contributor.pop(key)
```

Comment:
`KeywordSearchResponse.to_camel_case` is noisy. Hence the request to make it a function.